content_creator.py

```diff
@@ -61,8 +111,8 @@ def add_tools_to_bundle(bundle):
 
 # modify incident fields file to contain only `incidentFields` field (array)
 # from { "incidentFields": [...]} to [...]
-def convert_incident_fields_to_array():
-    scan_files = glob.glob(os.path.join(INCIDENT_FIELDS_DIR, '*.json'))
+def convert_incident_fields_to_array(incident_fields_dir=INCIDENT_FIELDS_DIR):
```

Comment:
we can remove this function completely as there is no more `incidentfields.json` file.